modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -177,6 +178,12 @@ public void testAddObjectAction() throws Exception {
 
 		// Add object actions
 
+		_objectDefinition.setEnableObjectEntryDraft(true);
+
+		_objectDefinition =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				_objectDefinition);
+
```

Comment:
I would put this code closer to its use.
(right after "// Add object entry as draft and update")

Reply:
Done!
Obs.: To do this, I had to make the `_publishCustomObjectDefinition` return an ObjectDefinition get the returned value [here](https://github.com/liferay-objects/liferay-portal/pull/2516/commits/90905d4320d55bacabb4506687a3e86c5e4ab624#:~:text=_objectDefinition%20%3D-,_publishCustomObjectDefinition,-()%3B).